src/main/groovy/com/github/benmanes/gradle/versions/updates/DependencyUpdatesReporter.groovy

```diff
@@ -130,6 +131,9 @@ class DependencyUpdatesReporter {
       case 'xml':
         reporter = new XmlReporter(project, revision, gradleReleaseChannel)
         break
+      case 'html':
+        reporter = new HtmlReporter(project, revision, gradleReleaseChannel)
+        break
```

Comment:
Can you update the readme with this new feature? :)

Reply:
On it. Though adding an image/actual html is a bit out of reach for me so I'm just adding a description. Feel free to critique.

Reply:
Next commit.